lesson9/src/phonebook/PhoneBook.java

```diff
@@ -0,0 +1,33 @@
+package phonebook;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PhoneBook {
+    private ArrayList<Record> records = new ArrayList<Record>();
+    public PhoneBook() {
+    }
+
+    public void add(Record record){
+        records.add(record);
+    };
```

Comment:
purely designed code, this method creates more code that executes. please delete add method and use the internal list method to add